scripts/process-podspecs.sh

```diff
@@ -41,8 +41,8 @@ lint() {
   if [ "${SUBSPEC:-}" ]; then
     local SUBSPEC_OPT="--subspec=$SUBSPEC"
   fi
-  pod lib lint $SUBSPEC_OPT $POD_LINT_OPT
-  pod lib lint $SUBSPEC_OPT $POD_LINT_OPT --use-libraries
+  bundle exec pod lib lint $SUBSPEC_OPT $POD_LINT_OPT
+  bundle exec pod lib lint $SUBSPEC_OPT $POD_LINT_OPT --use-libraries
```

Comment:
nit on these 2 lines:

https://github.com/koalaman/shellcheck/wiki/SC2086

consider adding double quotes accordingly

Reply:
@fkgozali will do in this one, but this should be only for `$SUBSPEC_OPT`, right? AFAIU `$POD_LINT_OPT` is *meant to* be expanded as separated entries, otherwise it will fail.

Reply:
btw, if that's the case then I also need to change:

```sh
process() {
  cd $1
  if [ -z "$SKIP_LINT" ]; then
    lint $2 # <--- HERE
  fi
```

Reply:
hmm I think that's for both actually (see if that works)

For $2, I think that's reasonable too, ref: https://github.com/koalaman/shellcheck/wiki/SC2086#correct-code

Reply:
got it, actually I added a new commit that fixes all the issues spotted by shellcheck. It's the last commit.

Reply:
Thank you!